src/utils/safeJSONParse.ts

```diff
@@ -0,0 +1,10 @@
+export const safeJSONParse = (str: string) => {
```

Comment:
Can you explain/comment the pattern please?
Since you later `throw` when calling `isValidJSON` it seems like it's just hiding the original error.

Reply:
I've added some comments, but in general we don't care about the error message as we show a generic one.

Reply:
I was more thinking of preserving the original error for debugging purposes.
Do you have a link for this pattern?
The access by array to validate is offsetting.
It implies that to use function to access the parse result you need to write:
`safeJSONParse(str)[1]`.
Would something like this work ? 
```
try {
 return JSON.parse(data)
} catch (err) {
 return false
}
```